src/Nethermind/Nethermind.Network.Discovery/IDiscoveryConfig.cs

```diff
@@ -111,4 +111,7 @@ public interface IDiscoveryConfig : IConfig
 
     [ConfigItem(DefaultValue = "0.05")]
     float DropFullBucketNodeProbability { get; set; }
+
+    [ConfigItem(Description = "Limit number of outgoing discovery message per second.", DefaultValue = "100", HiddenFromDocs = true)]
```

Comment:
Maybe document it more, what would be reasonable limit if issues?

Reply:
I don't know what works. Can you try on your end?

Reply:
Better late than never, but I think it works fine